ApolloExtra.lua

```diff
@@ -2664,55 +2914,55 @@ OlympusLootData.ApolloUpgrade = {
 				PreEventFunctionName = "BoonInteractPresentation", PreEventFunctionArgs = { PickupWait = 1.0, },
 				RequiredTextLines = { "ApolloFirstPickUp" },
 				RequiredFalseTextLines = { "OlympianReunionQuestComplete" },
-				{ Cue = "/VO/Apollo_0014",
+				{ Cue = "/VO/Apollo_0005",
 					StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
-					Text = "Hey Zagzag, are you still down there? Do you mind when I call you Zagzag? Because if you do, you better come and tell me face to face." },
+					Text = "ApolloMiscPickup04" },
 			},
 			ApolloMiscPickup05 =
 			{
                 Name = "ApolloMiscPickup05",
 				PreEventFunctionName = "BoonInteractPresentation", PreEventFunctionArgs = { PickupWait = 1.0, },
 				RequiredTextLines = { "ApolloFirstPickUp" },
-				{ Cue = "/VO/Apollo_0015",
+				{ Cue = "/VO/Apollo_0006",
 					StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
-					Text = "Are you still down there, Zagzag? I'll give you my help once more. Don't waste it." },
+					Text = "ApolloMiscPickup05" },
 			},
 			ApolloMiscPickup06 =
 			{
                 Name = "ApolloMiscPickup06",
 				PreEventFunctionName = "BoonInteractPresentation", PreEventFunctionArgs = { PickupWait = 1.0, },
 				RequiredTextLines = { "ApolloFirstPickUp" },
-				{ Cue = "/VO/Apollo_0016",
+				{ Cue = "/VO/Apollo_0007",
 					StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
-					Text = "What did I miss since last time, anything of interest, {#DialogueItalicFormat}hm{#PreviousFormat}? You'll have to fill me in once you break free, dearest." },
+					Text = "ApolloMiscPickup06" },
 			},
 			ApolloMiscPickup07 =
 			{
                 Name = "ApolloMiscPickup07",
 				PreEventFunctionName = "BoonInteractPresentation", PreEventFunctionArgs = { PickupWait = 1.0, },
 				RequiredTextLines = { "ApolloFirstPickUp" },
-				{ Cue = "/VO/Apollo_0017",
+				{ Cue = "/VO/Apollo_0008",
 					Emote = "PortraitEmoteCheerful",
 					StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
-					Text = "Well, look at you, there, love! So well-prepared to take on any danger that may come! But, I must wonder, are you well-prepared for {#DialogueItalicFormat}this{#PreviousFormat}?" },
+					Text = "By all the divines! You are still going at it, Zagzag. I see you are well determined to come meet us. Wish you the best!" },
```

Comment:
"By all the divines! You are still going at it, Zagzag? I see you are well determined to meet us at Olympus. Wish you the best!"